node/src/service.rs

```diff
@@ -238,23 +265,34 @@ async fn build_relay_chain_interface(
 	}
 }
 
-// We use this as the RuntimeApi definition in the following function so that we don't need to specify so many generic traits.
-// This might have to change if we want to use this function for other runtimes.
-use amplitude_runtime::RuntimeApi as AmplitudeRuntimeApi;
+type FullDepsOf<RuntimeApi, Executor> = FullDeps<
```

Comment:
Just a type that is used in the function `start_node_impl`, but is so complex that I wanted to define here.